es-design-system/pages/organisms/es-file-upload.vue

```diff
@@ -0,0 +1,174 @@
+                </ds-responsive-table-row>
+            </ds-responsive-table>
+        </div>
+        <div class="mb-450">
+            <h2>
+                EsFileInput event listeners
+            </h2>
+            <ds-responsive-table>
+                <ds-responsive-table-row
+                    v-for="eventListener in fileUploadEventListeners"
+                    :key="eventListener.name">
+                    <dl>
+                        <dt>
+                            Name
+                        </dt>
+                        <dd>
+                            <code>{{ eventListener.name }}</code>
+                        </dd>
+                    </dl>
+                    <dl>
+                        <dt>
+                            Payload
+                        </dt>
+                        <dd>
+                            <code>{{ eventListener.payload }}</code>
+                        </dd>
+                    </dl>
+                    <dl>
+                        <dt>
+                            Description
+                        </dt>
+                        <dd>
+                            {{ eventListener.description }}
+                        </dd>
+                    </dl>
+                </ds-responsive-table-row>
+            </ds-responsive-table>
+        </div>
+        <ds-doc-source
+            :comp-code="compCode"
+            comp-source="es-vue-base/src/lib-components/EsFileInput.vue"
+            :doc-code="docCode"
+            doc-source="es-design-system/pages/molecules/es-file-input.vue" />
+    </div>
+</template>
+<script>
+export default {
+    name: 'EsFileUploadDocs',
+    data() {
+        return {
+            compCode: '',
+            docCode: '',
+            uploadUrls: [],
+            fileUploadProps: [{
+                name: 'uploadUrls',
+                default: '[]',
+                description: 'A list of URLs to upload files to. When the number of files ready to upload matches '
+                + 'the number of URLs, the component will begin uploading.',
+            },
+            {
+                name: 'fileTypes',
+                default: 'None',
+                description: 'An array of accepted mime types for a file. If no argument passed, all file types are '
+                + 'accepted. These mime types follow the IANA Media Types.',
+            },
+            ],
+            fileUploadEventListeners: [{
+                name: '@readyToUpload',
+                payload: 'Number',
+                description: 'Called when the component is ready to upload files. The payload is the number of files '
+                + 'that are ready to upload.',
+            },
+            {
+                name: '@removeFile',
+                payload: 'String',
+                description: 'Called when the user clicks the remove button on a file. The payload is the name of the '
+                + 'file that was removed.',
+            }],
+        };
+    },
```

Comment:
nit: do you want to add a row for slots in this component?

Reply:
Added!